parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -72,6 +77,11 @@ def __init__(self,
              caps the maximum number of workers that can be launched.
              default: infinity
 
+        prefetch_capacity : int
+             Number of tasks that could be prefetched over available worker capacity. Default:100.
+             When there are a few tasks (<100) or when tasks are long running, this option should
+             be set to 0 for better load balancing.
+
```

Comment:
It's not immediately clear what to me the difference between `max_queue_size` and `prefetch_capacity`, so I'm going to assume that that isn't clear to other users. I'll have a bit more of a poke into the code to try to get my head round it.

Reply:
One of these are redundant. `max_queue_size` is unnecessary to expose to the user. It is the actual capacity that we'll have on the manager side.
I'm removing this as a configurable option.

Reply:
@benclifford please check the updated code and see if that resolves it. I've updated the code to have max_queue_size be the prefetch_capacity + worker_count.

Reply:
that makes sense